backend/src/main/java/de/dennismaas/thegramfworkingtitle/dto/AddPlaceDto.java

```diff
@@ -5,6 +5,7 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
```

Comment:
There is a lot of information in your model, maybe use the `@Builder` annotation to structure your models in tests.

Reply:
> There is a lot of information in your model, maybe use the `@Builder` annotation to structure your models in tests.



> There is an odd looking dependency in your `pom.xml`. Maybe this is in conflict with the test dependency you already have.
> 
> ```java
> <dependency>
> 			<groupId>org.springframework.boot</groupId>
> 			<artifactId>spring-boot-starter-test</artifactId>
> 			<scope>test</scope>
> 			<exclusions>
> 				<exclusion>
> 					<groupId>org.junit.vintage</groupId>
> 					<artifactId>junit-vintage-engine</artifactId>
> 				</exclusion>
> 			</exclusions>
> 		</dependency>
> ```

this is from stackoverflow so it´s not a problem

> junit-vintage-engine is used for running JUnit 4 tests; junit-jupiter-engine for JUnit 5 tests.
> 
> Presumably since you'll be writing only JUnit 5 tests for a new Spring Boot project, the vintage engine won't be needed, hence the default dependency exclusion in the POM.
> 
> Reference:
> 
> https://junit.org/junit5/docs/current/user-guide